tests/integration/ceph_csi_test.go

```diff
@@ -44,6 +44,12 @@ func runCephCSIE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suit
 		createCSIStorageClass(k8sh, s, namespace)
 		createCSIRBDTestPod(k8sh, s, namespace)
 		createCSICephFSTestPod(k8sh, s, namespace)
+
```

Comment:
Due to #3517 this new deletion code path isn't being hit because we're stuck on `isCSIRunning()` above. As a quick fix to add to this PR as well, could you take the suggestion from #3517 and replace `isCSIRunnig()`? The call to `isCSIRunning()` likely doesn't work anymore since the change was made to delay starting the flex and csi driver until the cluster CR is created.
```
if !k8sh.VersionAtLeast("1.14.0") {
  t.Skip()
}
```

Reply:
@travisn, as we are not creating any resource if csi pods are not running, this looks fine.
do I need to do anything else or missing something here?

Reply:
added code to skip csi tests if  Kube version is less than `1.13.0`